extensions/wikia/Answers/SpecialCreateDefaultQuestionPage.php

```diff
@@ -1,58 +1,108 @@
 <?php
 class CreateQuestionPage extends UnlistedSpecialPage {
 
+	const QS_KEY_QUESTION = 'questiontitle';
+
 	function __construct(){
 		parent::__construct("CreateQuestionPage");
 	}
 
-	function execute( $question ){
-		global $wgRequest, $wgOut, $wgUser;
+	function execute( $question ) {
+		if ( $this->getRequest()->wasPosted() ) {
+			return $this->createQuestion();
```

Comment:
Neither `createQuestion` nor `questionPage` return values, so there's nothing to return here or on L15 from them.  It would be better to move the `return` statement separately to the next line here and remove it altogether for L15.